Feature/odd 880 update logs cleanup

deoyani opened the pull request:
Updated and cleaned up code.
Added comments for functions.
Commented out extra log messages.
Updated logs level to 'INFO' by removing log level 'DEBUG' to reduce the size of logs per request.

RayPlante commented on src/main/java/gov/nist/oar/rmm/repositories/impl/CustomRepositoryImpl.java:
```diff
@@ -48,157 +48,131 @@
-	public Document find(Map<String,String> params){	
+	public Document find(Map<String, String> params) {
+
+//		logger.info("Find record with given parameters:" + params);
```
When the client requests a search that results in this function getting called, is the request logged with the parameters anywhere else?  If not, perhaps this line should be retained.

deoyani commented:
- Updated the logging level in GlobalExceptionHandler.

- Updated logs in SearchController to capture user request and parameters.

- Corrected error message in KeywordException

- CustomRepository updated to remove logging messages which are getting captured at controller level.

deoyani commented:
@RayPlante  Thanks for catching it, I updated the naming in logs.
